.github/workflows/build.yml

```diff
@@ -0,0 +1,13 @@
+name: Build
+on: [pull_request, workflow_dispatch]
```

Comment:
When we run this action on a PR, does it automatically target the source branch when running this? Saw this parameter (https://github.com/actions/checkout#checkout-a-different-branch) in the actions and wondering if we would need this, or if github actions are smart enough to run commands on the source branch rather than on main

Reply:
Actually reading a different part of the docs: 
```
    # The branch, tag or SHA to checkout. When checking out the repository that
    # triggered a workflow, this defaults to the reference or SHA for that event.
    # Otherwise, uses the default branch.
    ref: ''
```
So it seems like it would automatically use the source branch! Neat

Reply:
Yup - I think we're good on this.